Dockerfile.testing

```diff
@@ -3,7 +3,7 @@ FROM ubuntu:16.04
 MAINTAINER Bartek Nowotarski <bartek@stellar.org>
 
 ENV STELLAR_CORE_VERSION 15.0.0-40
-ENV HORIZON_VERSION 1.10.1-89
+ENV HORIZON_VERSION 1.11.0-90
```

Comment:
Just curious, is the `90` a "build number," whereas `1.11.0` is the "version"?

In other words, the "next version" could be anything, but it will always end with `-91`?

Reply:
yes